src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -994,7 +994,7 @@ public function getScheme()
      * If your reverse proxy uses a different header name than "X-Forwarded-Port",
      * configure it via "setTrustedHeaderName()" with the "client-port" key.
      *
-     * @return string
+     * @return int
```

Comment:
This can be a string in case of the request gets the port from the server bag.
I'm not sure that is bug of this method, but even if not it should be `string|int` at least.